easyrest/views/default.py

```diff
@@ -1,3 +1,7 @@
+"""Default controler for / route. Created by cookiecutter.
```

Comment:
I do not think you still need it. Backend will not serve any pages, only reply on REST requests. You may remove this view and corresponding route.